mypy_django_plugin/lib/helpers.py

```diff
@@ -173,16 +173,22 @@ def get_class_fullname(klass: type) -> str:
 def get_call_argument_by_name(ctx: Union[FunctionContext, MethodContext], name: str) -> Optional[Expression]:
     """
     Return the expression for the specific argument.
-    This helper should only be used with non-star arguments.
```

Comment:
This suggests we shouldn't do this - not sure if that is still applicable (or means something else)

Reply:
I guess maybe it means *args rather than **kwargs, so this comment should go back in